### PR TITLE
Add my models

### PR DESCRIPTION
**Pull request by Kim2091**
Adds my newer models

### CODE REVIEW

**Comment by Kim2091** [comment]
(Also added RPLKSR Dysample arch)

File: data/architectures.json [hunk 1]
```diff
@@ -146,6 +146,13 @@
             "onnx"
         ]
     },
+    "realplksr-dysample": {
+        "name": "RealPLKSR_dysample",
+        "input": "image",
+        "compatiblePlatforms": [
+            "pytorch"
+        ]
+    },
```
Comment (RunDevelopment) [inline data/architectures.json:155]: Why did you make this a new arch? Isn't dynsample just a hyperparameter for RealPLKSR?

Or does ONNX not support dynsample? In that case, I think a new arch would be fine.

Comment (Kim2091) [reply data/architectures.json:155]: You're right, it is a hyperparameter. I was thinking of it in terms of compatibility, my bad.

To my knowledge it does support ONNX, but i can't be 100% sure about that. I'm asking someone on EE right now

Comment (Kim2091) [reply data/architectures.json:155]: It supports statix onnx only. In this case, would you like me to remove the additional arch? @RunDevelopment

**Comment by RunDevelopment** [comment]
Sorry for the wait.

**Review by RunDevelopment — APPROVED** [review]

**Pull request merged**